src/components/Comments.jsx

```diff
@@ -19,6 +21,20 @@ function Comments({ article_id }) {
       .catch((err) => console.log(err));
   }, [article_id]);
 
+  const handleDeleteComment = (comment_id) => {
+    setIsDeletingCommentId(comment_id);
+    deleteComment(comment_id)
+      .then(() => {
+        setComments(
+          comments.filter((comment) => comment.comment_id !== comment_id)
+        );
+      })
+      .catch((err) => console.log(err))
```

Comment:
If there is an error currently nothing happens. Ideally we should in some way display this message so the user knows that something has gone wrong.